controllers/activemqartemis_reconciler.go

```diff
@@ -2200,10 +2214,14 @@ func configPodSecurity(podSpec *corev1.PodSpec, podSecurity *brokerv1beta1.PodSe
 	}
 }
 
-func determineImageToUse(customResource *brokerv1beta1.ActiveMQArtemis, imageTypeName string) string {
+func determineImageToUse(customResource *brokerv1beta1.ActiveMQArtemis, imageTypeName string) (string, error) {
 
 	imageName := ""
-	compactVersionToUse := determineCompactVersionToUse(customResource)
+	compactVersionToUse, err := determineCompactVersionToUse(customResource)
```

Comment:
I am not seeing the value of error handling here.... seems empty string is the default in the case of error or unknown.
if we drop the error no use computing it.

Reply:
yep, make sense.